core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -35,13 +35,15 @@ object ControllerConfig {
     ControllerConfig(
       statusUpdateIntervalMs = Option(100),
       resultUpdateIntervalMs = Option(1000),
-      monitoringIntervalMs = Option(3000)
+      monitoringIntervalMs = Option(3000),
+      skewDetectionIntervalMs = Option(3000)
```

Comment:
can we move those to config files please.

Reply:
Sure. will do that change too in PR #1414 .